Sources/HaCWebsiteLib/Events/Event.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+protocol Event {
+    var title : String {get}
+    var time : DateInterval {get}
+    var tagLine : String {get}
```

Comment:
@Pinpickle here I've left out the `eventDescription` from the protocol but I've put it in some implementations where it's needed instead. I have included a `tagLine` instead.